mycroft/tts/tts.py

```diff
@@ -355,9 +355,6 @@ def execute(self, sentence, ident=None, listen=False):
         try:
             self._execute(sentence, ident, listen)
         except Exception:
-            # If an error occurs end the audio sequence through an empty entry
-            self.queue.put(EMPTY_PLAYBACK_QUEUE_TUPLE)
-            # Re-raise to allow the Exception to be handled externally as well.
             raise
```

Comment:
Maybe the try-except can be removed completely when the except just re-raises the exception.